### DISCUSSION TITLE
VALUE function

### DISCUSSION THREAD

**Discussion by forrestlinSYD** [handsontable/hyperformula#1335] [Feature requests]
Hi team,
Would it be possible to support VALUE function in the future release?

**Comment by sequba** [comment]
Hi @forrestlinSYD, thanks for reaching out to us. `VALUE` function should be pretty easy to implement, but it's not necessary in most cases as HyperFormula automatically converts strings to numbers based on the context.

Do you have a specific use case that doesn't work with automatic string conversion and requires the `VALUE` function? Or it's just for compatibility with other spreadsheet programs?
